src/app/zones/views/__snapshots__/ZonesView.spec.ts.snap

```diff
@@ -888,6 +888,30 @@ exports[`ZonesView.vue renders snapshot when multizone 1`] = `
                                 cluster-1
                               </p>
                             </li>
+                            <li>
+                              <h4>
+                                status
+                              </h4>
+                              <p>
+                                <div
+                                  class="k-badge d-inline-flex k-badge-success k-badge-rounded"
+                                  tabindex="0"
+                                >
+                                  <div
+                                    class="k-badge-text truncate"
+                                  >
+                                    <div
+                                      class="k-badge-text truncate"
+                                    >
+                                      
+                                      online
+                                      
+                                    </div>
+                                  </div>
+                                  <!---->
+                                </div>
+                              </p>
+                            </li>
```

Comment:
We gained a `status` output here, but I don't see any mocks that changed, so whilst we might need to show `status` I'm guessing some logic changed somewhere to mean that this now shows?

Reply:
That’s right. The component clearly indicated that the status was meant to be shown (hence the table column for it). There was other code surrounding statuses, too. However, this was partially broken and resulted in the statuses never making it into the page. Specifically, the component’s `getEntity` method didn’t add the status to the `entity` object and so the table, while trying to display it, never had a value.

So, technically that’s a small bug fix.

Reply:
Gotcha, merge away!